init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
> This file is duplicated across repos, and trying to keep it "in sync" isn't going to be possible.

We aren't really keeping it in sync, rather we are following the design pattern that was put forth by corefx for initialize scripts